components/slack/slack.app.mjs

```diff
@@ -310,6 +331,12 @@ export default {
       default: true,
       optional: true,
     },
+    post_at: {
+      label: "Post At",
+      description: "Schedule the message to a specific moment. [See the docs here] (https://api.slack.com/messaging/scheduling)",
```

Comment:
should be: `...[See the docs here](https://api.slack.com/messaging/scheduling)`

without the space between `] (` so that it renders as a hyperlink in the UI